cpp-ch/local-engine/Parser/RelParsers/ReadRelParser.cpp

```diff
@@ -53,15 +56,35 @@ ReadRelParser::parse(DB::QueryPlanPtr query_plan, const substrait::Rel & rel, st
 {
     if (query_plan)
         throw DB::Exception(DB::ErrorCodes::LOGICAL_ERROR, "Source node's input plan should be null");
+
     const auto & read = rel.read();
-    if (isReadFromDefault(read))
```

Comment:
@baibaichen could you please review changs in ReadRelParser.* and SerializedPlanParser.*